src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System.Runtime.InteropServices.JavaScript.Tests.csproj

```diff
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
+<!--
```

Comment:
Lets delete the line if not needed any more

Reply:
this is mistake, thanks